as2_utilities/as2_external_object_to_tf/config/external_objects.yaml

```diff
@@ -4,22 +4,22 @@ objects:
     parent_frame: earth
     frame: object_0
     pose_topic: "/object_0/pose"
-  # - type: gps
-  #   parent_frame: earth
-  #   frame: object_1
-  #   gps_topic: "/object_1/sensor_measurements/gps"
-  #   azimuth_topic: "/object_1/gps/azimuth"
-  # - type: gps_static
-  #   parent_frame: earth
-  #   frame: object_2
-  #   gps_pose:
-  #     lat: 52.171775999796274
-  #     lon: 4.416570248787041
-  #     alt: 50.520017601549625
-  #   orientation:
-  #     azimuth: 180.0
-  #     bank: 0.0
-  #     elevation: 0.0
+  - type: gps
```

Comment:
Why uncomment this?

Reply:
Before, it was the default config file, when gps data is set in the config file the node waits indefinitely for an origin request, thats why it was commented. Now it just serves as an example and a config file is required to execute the node, so it can be uncommented again as it won't be loaded by default.